src/main/java/beam/playground/point2point/PointToPointMain.java

```diff
@@ -0,0 +1,37 @@
+package beam.playground.point2point;
+
+import com.conveyal.r5.transit.TransportNetwork;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * Created by ahmar.nadeem on 5/28/2017.
+ */
+public class PointToPointMain {
+
+    private static final Logger LOG = LoggerFactory.getLogger(PointToPointMain.class);
+
+    public static void main(String directoryPath[]) {
+
+        final boolean inMemory = false;
+
+        File dir = new File(directoryPath[0]);
```

Comment:
Shouldn't check if there is any argument, it could cause potentially null pointer exception.